source/scss/_pf-component-alert.scss

```diff
@@ -0,0 +1,38 @@
+//
+// Alerts
+// --------------------------------------------------
+
+.alert {
+  padding-left: $spacer;
+  padding-right: $spacer;
+  display: flex;
+  align-items: center;
+
+  .alert-link {
+    color: $link-color;
+    &:hover {
+      color: $link-hover-color;
+    }
+  }
+}
+
+.alert-icon {
+  font-size: $alert-close-font-size;
+  margin-right: $spacer;
+}
+
+.alert-text {
+  flex-grow: 1;
+}
+
+.alert-dismissible .close {
```

Comment:
I know we will have to do the nesting to position the "x".
The close component will be reuse by other components. Can we create the close element as a component and the include it in other places? Define it once with as much patternfly styles as we can and then add here only the  styles needed for alerts.
also, can we nest `.close` so it's more clear?